client/account-recovery/reset-password/reset-password-form/index.jsx

```diff
@@ -93,39 +110,51 @@ export class ResetPasswordFormComponent extends Component {
 						'select one of these options and follow the instructions.'
 					) }
 				</p>
-				<Card>
-					<FormFieldset className="reset-password-form__field-set">
-						<FormLegend className="reset-password-form__legend">
-							{ translate( 'How would you like to reset your password?' ) }
-						</FormLegend>
-						{ resetOptions.map( ( { email, sms, name }, index ) => (
-							<ResetOptionSet
-								key={ index }
-								email={ email }
-								sms={ sms }
-								name={ name }
-								displayStrings={ this.getOptionDisplayStrings( name ) }
-								disabled={ isSubmitting }
-								onOptionChanged={ this.onResetOptionChanged }
-								selectedResetOption={ selectedResetOption }
-							/>
-						) ) }
-					</FormFieldset>
-					<Button
-						className="reset-password-form__submit-button"
-						onClick={ this.submitForm }
-						disabled={ ! isPrimaryButtonEnabled }
-						primary>
-						{ translate( 'Continue' ) }
-					</Button>
-				</Card>
-			</div>
+				<FormFieldset className="reset-password-form__field-set">
+					<FormLegend className="reset-password-form__legend">
+						{ translate( 'How would you like to reset your password?' ) }
+					</FormLegend>
+					{ resetOptions.map( ( { email, sms, name }, index ) => (
+						<ResetOptionSet
+							key={ index }
+							email={ email }
+							sms={ sms }
+							name={ name }
+							displayStrings={ this.getOptionDisplayStrings( name ) }
+							disabled={ isRequesting }
+							onOptionChanged={ this.onResetOptionChanged }
+							selectedResetOption={ pickedMethod }
```

Comment:
No reason. Just something came to my mind when I wrote it :)
Updated by 8dbb8626c1aa3a3ab39597b37196b54d2f8716e1.